app/models/participant_declaration/npq.rb

```diff
@@ -2,4 +2,18 @@
 
 class ParticipantDeclaration::NPQ < ParticipantDeclaration
   include RecordDeclarations::NPQ
+
+  scope :for_course, ->(course_identifier) { where(course_identifier: course_identifier) }
+  scope :for_lead_provider_and_course, ->(cpd_lead_provider, course_identifier) { for_lead_provider(cpd_lead_provider).for_course(course_identifier) }
```

Comment:
do we need a compound scope, can't we just chain the building blocks in situ? otherwise we'll have many permutations of compound scopes